src/app/ApplicationTemplate.UWP/Views/Styles/Controls/Button.xaml

```diff
@@ -254,6 +254,95 @@
 		</Setter>
 	</Style>
 
+	<Style x:Key="SettingsButtonStyle"
+		   TargetType="Button">
+		<Setter Property="Background"
+				Value="Transparent" />
+		<Setter Property="Foreground"
+				Value="{StaticResource MaterialOnSurfaceBrush}" />
```

Comment:
I see you're using OnSurface here, but the button is actually used on of top of the Background color from what I can see in SettingsPage.xaml.